tools/lab_dispatch/build_install_deb.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
```

Comment:
This might be controversial but I will go ahead and ask it: what is the advantage of this being a Python program instead of a bash script? The core of the code runs bash commands so I personally think it would actually be shorter and more readable if this were a bash script.

Reply:
I still think the Python script is unreadable but it's your call.